### PR TITLE
feat(pds-ember): add Tabbed Nav

### PR DESCRIPTION
**Pull request by CITguy**
Branched off of #42 

JIRA: PDS-177

## Screenshots
examples - tabnav

### CODE REVIEW

File: packages/pds-ember/addon/components/pds/tab-nav/index.hbs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<nav
+  class="pds-tabNav"
```
Comment (jesdavpet) [inline packages/pds-ember/addon/components/pds/tab-nav/index.hbs:2]:
```suggestion
  class="pds-tab-nav"
```

File: packages/pds-ember/app/styles/pds/components/tab-nav/__config.scss [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+    @include Link.whenCurrent {
+      @include Tab.current-appearance;
+    }
+  }
+}
+
+@mixin apply {
+  /* [debug] #{$_module}@apply */
+  .pds-tabNav {
```
Comment (jesdavpet) [inline packages/pds-ember/app/styles/pds/components/tab-nav/__config.scss:35]:
```suggestion
  .pds-tab-nav {
```

**Review by jesdavpet — COMMENTED** [review]
Have we determined CSS conventions / methodology on the cloud team @CITguy?

🤔 Are we using ["BEM" or "OOCSS" methodologies](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Organizing) ... or maybe something else?

My nitpicks/suggestions here WRT using `kebab-case` class names could be a little off-base, depending.

**Review by jesdavpet — COMMENTED** [review]
Will all of these debugs remain in the codebase long-term / permanently?

👀 I looked through a bunch of the existing `*.scss` files in this repository ... and it seems like a mixed bag so far ... some existing files outside of this PR have debug statements, others do not.

File: packages/pds-ember/app/styles/pds/components/tab-nav/_top.scss [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+@use "../link/__config" as Link;
+
+$borderColor: theme.$borderColor;
+$borderWidth: 1px;
+
+@mixin layout {
+  display: flex;
+  border-bottom: $borderWidth solid $borderColor;
+
+  // FIXME: how should this [over]flow with constrained dimensions?
```
Comment (jesdavpet) [inline packages/pds-ember/app/styles/pds/components/tab-nav/_top.scss:14]: Is there either a design spec for this edge case, or an existing JIRA ticket to reference?

My concern would be that this important question could get lost, if it isn't captured elsewhere.

File: packages/pds-ember/app/styles/pds/components/index.scss [hunk 4]
```diff
@@ -19,4 +19,5 @@
 @use "select";
 @use "sidebar";
 @use "skip-link"; // TODO: move to "link/skip-link" as variant
```
Comment (jesdavpet) [inline packages/pds-ember/app/styles/pds/components/index.scss:21]: Is there either a design spec for this edge case, or an existing JIRA ticket to reference?

My concern would be that this important question could get lost, if it isn't captured elsewhere.

**Review by jesdavpet — APPROVED** [review]
I really ❤️ the attention to detail in the documentation here.
Playing with the Vercel preview build made this a pleasure to review! 😃
👌 That's a really nice touch.

I did have a couple questions on this, but didn't want to make any of them requested changes, since I'm not certain of this team's conventions and practices yet. 🙏 Please take those with a grain of salt.

File: packages/pds-ember/addon/components/pds/tab-nav/index.hbs (hunk 1 above)
Comment (CITguy) [reply packages/pds-ember/addon/components/pds/tab-nav/index.hbs:2]: Class names follow the convention [documented in the wiki](https://github.com/hashicorp/structure/wiki/Stylesheet-Conventions#class-names)

File: packages/pds-ember/app/styles/pds/components/link/tab/__config.scss [hunk 5]
```diff
@@ -0,0 +1,26 @@
+/// Styles for tab links (above content) (See pds/components/tab-nav)
+$_module: "PDS.Components.Link.Tab";
+/* --- [debug] CONFIG: #{$_module} --- */
```
[the remark this answers is not in the document]
Comment (CITguy) [reply packages/pds-ember/app/styles/pds/components/link/tab/__config.scss:3]: HTML comments will exist in the dev CSS, but get removed in prod CSS as a result of minification.

File: packages/pds-ember/app/styles/pds/components/tab-nav/_top.scss (hunk 3 above)
Comment (CITguy) [reply packages/pds-ember/app/styles/pds/components/tab-nav/_top.scss:14]: See https://www.figma.com/file/B7yd8loYS7nA2TiQcJ8UtV?node-id=3771:6348#48254074

File: packages/pds-ember/app/styles/pds/components/index.scss (hunk 4 above)
Comment (CITguy) [reply packages/pds-ember/app/styles/pds/components/index.scss:21]: captured via PDS-195

File: packages/pds-ember/app/styles/pds/components/tab-nav/_top.scss (hunk 3 above)
Comment (CITguy) [reply packages/pds-ember/app/styles/pds/components/tab-nav/_top.scss:14]: also captured in PDS-196

**Comment by CITguy** [comment]
@jesdavpet, thanks for your feedback. I've captured some of it in JIRA and hopefully clarified some of your questions, too.

**Pull request merged**